### DISCUSSION TITLE
Is there a way to run a command after prisma generate?

### DISCUSSION THREAD

**Discussion by zomars** [prisma/prisma#11274] [Q&A] [answered]
Some thing like a post generate event. Right know we are using external prisma generators but each one use their own formatting style. If we could use a `post generate` event where we can just run something like `prettier --write ./prisma` that would be awesome.

**Comment by austincrim** [comment, marked as answer]
Hey @zomars 👋 ,

This is not possible today but it would be great if you could open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) with more details about your use case. Our product and engineering teams will take it from there 👍.

**Comment by CarterRoll** [comment]
Is this a feature yet? I use https://github.com/chrishoermann/zod-prisma-types to generate types to programmatically determine the format of the object to put in the database. However, it doesn't have an option to coerce numbers or booleans. So, every time I generate the Prisma schema, I need to go into the file generated from that library and update all of my numbers and booleans to coerce.

This is an easy process to forget to do, so being able to apply a script to run after generation to automate this process would be extremely convenient.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!

**Comment by fogx** [comment]
also looking for a solution to this. @zomars @CarterRoll did you find anything?

**Reply by zomars** [reply]
I've added generated files to prettier and git ignore files and moved on.
